Easydict/NewApp/LanguagePreference/EZLocalizedBundle.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  EZLocalizedBundle.swift
+//  Easydict
+//
+//  Created by choykarl on 2024/3/23.
+//  Copyright © 2024 izual. All rights reserved.
+//
+
+import Foundation
+
+class EZLocalizedBundle: Bundle {
```

Comment:
Same as `I18nHelper`, should not have the namespace prefix `EZ` in Swift.

Reply:
you are right